mayastor-test/e2e/node_disconnect/iscsi_drop/node_disconnect_iscsi_drop_test.go

```diff
@@ -30,7 +30,7 @@ func TestNodeLoss(t *testing.T) {
 }
 
 var _ = Describe("Mayastor node loss test", func() {
-	It("should verify behaviour when a node becomes inaccessible", func() {
+	It("should verify iscsi nexus behaviour when a node becomes inaccessible (dropped connection)", func() {
```

Comment:
Love how it is now explicit (and I'm expecting on failed test one doesn't have to go through code to get the details). :+1: I'm however a bit perplexed by `dropped connection`. The reason is we're running over TCP so it is just packets that are dropped (not the connection - it might even recover later when e.g. network problem we're testing here is fixed and e.g. packet loss stops). I found out that being explicit about what is being done - rather than the attempted outcome being tested - helps when test starts to fail as it is more explicit about how the environment is setup. What about:

```suggestion
	It("should verify iscsi nexus behaviour when a node becomes inaccessible (iptables DROP)", func() {
```